vowpalwabbit/core/src/reductions/cb/cb_algs.cc

```diff
@@ -134,96 +134,110 @@ void eval_finish_example(VW::workspace& all, cb& c, VW::example& ec)
   output_example(all, c, ec, ec.l.cb_eval.event);
   VW::finish_example(all, ec);
 }
-}  // namespace
 
```

Comment:
Nit:  // Struct to store options for v1 of cb_algs
Suggestion for convention for these structs?
for example options_<reduction>_v<version>?  so struct options_cb_algs_v1 ?

Reply:
`cb_algs_options_instance_v1` this is the current convention I have been using. I agree it could be shorter